pennylane/ops/qubit.py

```diff
@@ -1977,6 +1977,231 @@ def adjoint(self, do_queue=False):
         return QFT(wires=self.wires, do_queue=do_queue).inv()
 
 
+# =============================================================================
+# Quantum chemistry
+# =============================================================================
+
+
+class DoubleExcitation(Operation):
+    r"""DoubleExcitation(phi, wires)
+    Double excitation rotation.
+
+    .. math:: U(\phi) = \begin{bmatrix}
+                1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+                0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+                0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+                0 & 0 & 0 & \cos(\phi/2) & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & -\sin(\phi/2) & 0 & 0
+                & 0\\
+                0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+                0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+                0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+                0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+                0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+                0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 & 0\\
+                0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 & 0\\
+                0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0\\
+                0 & 0 & 0 & \sin(\phi/2) & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & \cos(\phi/2) & 0 & 0 & 0\\
+                0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0\\
+                0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0\\
+                0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1
+            \end{bmatrix}.
+
+
+    This operation performs a rotation in the two-dimensional subspace :math:`\{|1100\rangle,
+    |0011\rangle\}`. The name originates from a fermionic interpretation, where the transformation
+    from :math:`|1100\rangle` to `:math:`|0011\rangle` is interpreted as "exciting" two particles
+    from the first pair of qubits to the second pair of qubits.
+
+    **Details:**
+
+    * Number of wires: 4
+    * Number of parameters: 1
+    * Gradient recipe: Obtained from its decomposition in terms of the
+        :class:`~.DoubleExcitationPlus` and `:class:`~.DoubleExcitationMinus` operations
```

Comment:
Suggest adjusting the indentation and removing the extra leading backtick as in #1121 (recurring)
```suggestion
    * Gradient recipe: Obtained from its decomposition in terms of the
      :class:`~.DoubleExcitationPlus` and :class:`~.DoubleExcitationMinus` operations
```